classes/Pods.php

```diff
@@ -3441,7 +3441,7 @@ public function form ( $params = null, $label = null, $thank_you = null ) {
 		$thank_you = $params[ 'thank_you' ];
 		$fields_only = $params[ 'fields_only' ];
 
-		PodsForm::$form_counter++;
+		$form_counter = PodsForm::$form_counter ++;
```

Comment:
We try to avoid complex lines like these, each line should do one thing preferably. So ++ on one line, the $form_counter set on the next (or just use the PodsForm var down in form.php